product_docs/docs/pem/8/registering_database_server.mdx

```diff
@@ -8,7 +8,169 @@ redirects:
+
+## How to register a Postgres server
+There are three methods to register a server:
+
+  - [Use the pemworker utility to register a database server](#using-the-pemworker-utility-to-register-a-server) from the command line on the host where the agent you wish to bind is located.
+  - [Use automatic server discovery](#registering-a-server-with-automatic-server-discovery) in the PEM web application to automatically register Postgres servers located on the same host as an agent.
+  - [Manually register a Postgres server](#manually-registering-a-database-server) by adding all the details in the PEM web application.
+
+## What does registering a Postgres server do?
+
+Registering a server does two things:
+
+  - It adds the server to the server tree in the PEM web application allowing users to open connections from the web application to the server
+  - It binds the server to a particular agent, meaning that agent will open connections to the server to execute probes and scheduled jobs
+
+!!! Note Connections from the agent to the monitored server
+During agent registration, you are required to provide connection details for the agent to connect to the monitored server. As is normal for Postgres, you must specify the database to connect to. 
+However, the PEM Agent will monitor **every database accessible to the specified user**, not just the database specified.
+
+After registration, the agent opens a connection to the specified database. It uses this connection to determine which other databases it can connect to with the same user and then spawns one connection to each. 
+!!!
+
+## Using the pemworker utility to register a server
+
+You can use the pemworker utility to register a Postgres server. During registration, the pemworker utility binds the new server to the agent that resides on the system from which you invoked the registration command. To register a server:
+
+On a Linux host, use the command:
+
+ `pemworker --register-server`
+
+On a Windows host, use the command:
+
+ `pemworker.exe REGISTER-SERVER`
+
+Append command line options to the command string when invoking the pemworker utility. Follow each option with a corresponding value.
+
+| Option&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I did this hack on the advice of the docs team. Without it, the options get broken up over many lines in a horrible way.

Reply:
Sigh. This is partly why I objected to moving to pure markdown in the first place. It just doesn't handle formatting nearly as well as ReStructuredText.

There must be some other way to avoid breaking lines without risking HTML formatting going awry...